docs/user/azure/tested_instance_types_x86_64.md

```diff
@@ -15,16 +17,41 @@
 * `standardEADSv5Family`
 * `standardEASv4Family`
 * `standardEASv5Family`
+* `standardEBDSv5Family`
+* `standardEBSv5Family`
 * `standardEDSv4Family`
 * `standardEDSv5Family`
+* `standardEIADSv5Family`
+* `standardEIASv4Family`
+* `standardEIASv5Family`
+* `standardEIDSv5Family`
+* `standardEISv3Family`
+* `standardEISv5Family`
 * `standardESv3Family`
 * `standardESv4Family`
 * `standardESv5Family`
+* `standardFXMDVSFamily`
 * `standardFSFamily`
 * `standardFSv2Family`
 * `standardGSFamily`
+* `standardHBrsv2Family`
+* `standardHBSFamily`
+* `standardHCSFamily`
+* `standardLASv3Family`
 * `standardLSFamily`
 * `standardLSv2Family`
+* `standardLSv3Family`
+* `standardMDSMediumMemoryv2Family`
+* `standardMIDSMediumMemoryv2Family`
+* `standardMISMediumMemoryv2Family`
 * `standardMSFamily`
+* `standardMSMediumMemoryv2Family`
+* `StandardNCADSA100v4Family`
+* `Standard NCASv3_T4 Family`
+* `standardNCSv2Family`
 * `standardNCSv3Family`
-* `standardNPSFamily`
```

Comment:
standardNPSFamily not support ?

Reply:
supported, it is in line 54, not deleted.